### PR TITLE
Skeleton post for stats comm call

### PR DESCRIPTION
**Pull request by steffilazerte**
Skeleton post for community call on Stats

* [X] I have read the Content Guidelines.
* [X] I have read the Technical Guidelines.
* [X] I used or followed the R Markdown or Markdown template.
* [X] I have followed the Style Guide.
* [X] I created or updated my author metadata with correct folder name.
* [X] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [X] I have added the "tech notes" tag if this is a technote.
* [X] I ran `roblog::ro_lint_md()` on index.md (optional).
* [X] I ran `roblog::ro_check_urls()` on index.md (optional).
* [X] I ran a spell-check on index.md.

Ready for introduction!

### CODE REVIEW

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    keep_md: true
+---
+
+A week ago we held a [Community Call](/commcalls/feb2021-statsreview/) discussing rOpenSci Statistical Software Testing and Peer Review. 
+This call included speakers [Noam Ross](/author/noam-ross/) [Mark Padgham](/author/mark-padgham), [Anna Krystalli](/author/anna-krystalli), [Alex Hayes](/author/alex-hayes) and [John Sakaluk](/author/john-sakaluk). 
+
+{{< figure src = "speakers.png" class = "center" width = "600" alt = "Headshots of the moderator and four panelists">}}
+
+We decided to write this additional blog post to provide a ready reference and description of this community call, which introduced the system being developed for peer review of explicitly statistical software, along with a couple of the automated software tools for use by developers and reviewers of statistical software.
+After an introduction by Stefanie Butland, Anna Krystali gave an overview of the context and importance of our new tools from an editorial perspective.
```
Comment (noamross) [inline content/blog/2021-03-09-commcall-stats/index.md:33]:
```suggestion

After an introduction by Stefanie Butland, Anna Krystali gave an overview of the context and importance of our new tools from an editorial perspective.
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Noam Ross then introduced the statistical software review project, members of its advisory board, and the standards-based system which will be used to assess and review statistical software.
+Mark Padgham then briefly described the two main tools intended for use by developers and reviewers: the [`autotest` package](https://ropenscilabs.github.io/autotest/) for automated testing of software to ensure robust responses to unexpected inputs throughout development, and the [`srr` (**s**oftware **r**eview **r**oclets) package](https://ropenscilabs.github.io/srr/) for documenting within code itself how and where it complies with both general and category-specific standards for statistical software.
+
+The call then moved on to its main intention of being a "hands-on" demonstration of how these packages can be used in practice, by first passing to [John Sakaluk](https://www.psychology.uwo.ca/people/faculty/profiles/sakaluk.html), who has developed the [`dySEM` package](https://jsakaluk.github.io/dySEM/) for his own use, and would now like to refine and extend the package for more general use, ideally working towards submission to our peer-review system.
```
Comment (noamross) [inline content/blog/2021-03-09-commcall-stats/index.md:37]:
```suggestion
The call then moved on to a "hands-on" demonstration of how these packages can be used in practice.  [John Sakaluk](https://www.psychology.uwo.ca/people/faculty/profiles/sakaluk.html) showed `autotests`'s capabilites on his [`dySEM` package](https://jsakaluk.github.io/dySEM/). John developed `dySEM` for his own use, and would now like to refine and extend the package for more general use, ideally working towards submission to our peer-review system.
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+John described the usefulness of [`autotest`](https://ropenscilabs.github.io/autotest/) in explicitly revealing aspects of his code which could be improved for more general usage.
+
+> (cool quotes from John once captions are up ...)
+
+[Alex Hayes](https://www.alexpghayes.com/) then described his experiences from initial review of his [`fastadi` package](https://github.com/RoheLab/fastadi), and of the role standards can play in software improvement and assessment, noting in particular the usefulness of standards as contextual "touchpoints" for review.
```
Comment (noamross) [inline content/blog/2021-03-09-commcall-stats/index.md:42]:
```suggestion
[Alex Hayes](https://www.alexpghayes.com/) then described his experiences from initial review of his [`fastadi` package](https://github.com/RoheLab/fastadi), and of the role standards can play in software improvement and assessment, noting in particular the usefulness of standards as contextual "touchpoints" for review, and how the [`srr` package](https://ropenscilabs.github.io/srr/) tracks these standards through the development process
```

**Review by noamross — COMMENTED** [review]
Made some small tweaks, looks good! (Still needs a good pull-quote)

**Comment by steffilazerte** [comment]
Everything looks good to me!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

I'll post a tweet draft soon, and unless anyone has any more comments I think it's ready for publication tomorrow.

**Comment by steffilazerte** [comment]
Draft tweet:

---

[blog] rOpenSci Statistical Software Testing and Peer Review - Community Call Summary

☹️ Did you miss last week's community call? 

😀 Check out our summary!

https://ropensci.org/blog/2021/03/09/commcall-stats/

#rstats #SoftwarePeerReview

(1/2)

---

🎉 @&#8203;noamross, @&#8203;bikesrdata, @&#8203;annakrystalli, @&#8203;JohnSakaluk, @&#8203;alexpghayes, & @&#8203;steffilazerte summarize the call with direct links to the video and collaborative document  🎉

🙏 Thank you to all attendees and for the great questions we received! 

(2/2)

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+> one of the things that's really useful for me here as a self taught and kind of newbie developer is I find myself adding to my kind of package development list almost every time that I open it up in terms of wish listing new functionality. And what's really nice about this [`autotest` tool] is this can help me kind of set some targets for priority items just for kind of tightening up the programming of the existing functions.
+
+[Alex Hayes](https://www.alexpghayes.com/) then described his experiences from initial review of his [`fastadi` package](https://github.com/RoheLab/fastadi), and of the role standards can play in software improvement and assessment, noting in particular the usefulness of standards as contextual "touchpoints" for review.
+
+Here we've organized the video content by speakers and questions, including links to the specific time points in the [video](https://vimeo.com/518761488) as well as to questions and answers in the [collaborative document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/) that we include in every community call.
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:44]: I recommend putting this paragraph after heading `Summarizing ...`. When I first browsed the post, I went straight to that heading and thought the above para was "missing"

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+[Alex Hayes](https://www.alexpghayes.com/) then described his experiences from initial review of his [`fastadi` package](https://github.com/RoheLab/fastadi), and of the role standards can play in software improvement and assessment, noting in particular the usefulness of standards as contextual "touchpoints" for review.
+
+Here we've organized the video content by speakers and questions, including links to the specific time points in the [video](https://vimeo.com/518761488) as well as to questions and answers in the [collaborative document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/) that we include in every community call. 
+We hope that by preparing this summary, more people will be able to benefit from this information.
+
+
+## Summarizing the community call
+
+### Speakers
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:50]: omit caps in all "titles" here I think e.g. The editorial perspective

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+- **John Sakaluk - Using autotest on a package-in-development** ([video](https://vimeo.com/518761488#t=23m34s))
+
+- **Alex Hayes - Using srr while preparing a package for review** ([video](https://vimeo.com/518761488#t=31m23s))
+
+- **Anna Kyrstalli - Moderates Questions** ([video](https://vimeo.com/518761488#t=38m47s))
+
+
+### Questions
+
+- (Anna) Suggestion of RStudio addin for srr ([video](https://vimeo.com/518761488#t=38m47s))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:67]:
```suggestion
- (Anna Krystalli) Suggestion of RStudio addin for srr ([video](https://vimeo.com/518761488#t=38m47s))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Anna) Could we get more detail on what autotest looks for? ([video](https://vimeo.com/518761488#t=39m45s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.so3v8xc34myn))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:69]:
```suggestion
- (Anna Krystalli) Could we get more detail on what autotest looks for? ([video](https://vimeo.com/518761488#t=39m45s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.so3v8xc34myn))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Steffi) Is autotest applicable to all packages or just statistical packages? ([video](https://vimeo.com/518761488#t=43m11s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.4th2qtdafspr))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:71]:
```suggestion
- (Steffi LaZerte) Is autotest applicable to all packages or just statistical packages? ([video](https://vimeo.com/518761488#t=43m11s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.4th2qtdafspr))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Kieran) What is missing from these package for assessing statistical packages? ([video](https://vimeo.com/518761488#t=44m43s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.jk4fef3ixoga))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:73]:
```suggestion
- (Kieran Martin) What is missing from these package for assessing statistical packages? ([video](https://vimeo.com/518761488#t=44m43s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.jk4fef3ixoga))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Joss) We are just beginning to re-code some existing packages - so have the advantage of starting with a blank sheet AND a good specification AND well tested code snippets. We're new to this community - what advice would give on both engagement and coding? ([video](https://vimeo.com/518761488#t=49m07s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.fvjzqwqcf88u))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:75]:
```suggestion
- (Joss Langford) We are just beginning to re-code some existing packages - so have the advantage of starting with a blank sheet AND a good specification AND well tested code snippets. We're new to this community - what advice would give on both engagement and coding? ([video](https://vimeo.com/518761488#t=49m07s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.fvjzqwqcf88u))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+- (John) Regarding representation in the program, do we have any sense where groups of folks are over/under-represented within specific areas? ([video](https://vimeo.com/518761488#t=53m27s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.xnstfjt1cjg7))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:77]:
```suggestion
- (John Sakaluk) Regarding representation in the program, do we have any sense where groups of folks are over/under-represented within specific areas? ([video](https://vimeo.com/518761488#t=53m27s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.xnstfjt1cjg7))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Rafael) What is the vision for applying these srr standards to existing (previously published) rOpenSci packages ([video](https://vimeo.com/518761488#t=55m40s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.78ydzxagwubi))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:79]:
```suggestion
- (Rafael Pilliard Hellwig) What is the vision for applying these srr standards to existing (previously published) rOpenSci packages ([video](https://vimeo.com/518761488#t=55m40s) | [document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.78ydzxagwubi))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+- (Charles) Does autotest take into account dependencies and dependency changes? ([document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.s52f459a2yp1))
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:81]:
```suggestion
- (Charles Sweetland) Does autotest take into account dependencies and dependency changes? ([document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/edit#heading=h.s52f459a2yp1))
```

File: content/blog/2021-03-09-commcall-stats/index.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+  html_document:
+    keep_md: true
+---
+
+A week ago we held a [Community Call](/commcalls/feb2021-statsreview/) discussing rOpenSci Statistical Software Testing and Peer Review. 
+This call included speakers [Noam Ross](/author/noam-ross/), [Mark Padgham](/author/mark-padgham), [Anna Krystalli](/author/anna-krystalli), [Alex Hayes](/author/alex-hayes), and [John Sakaluk](/author/john-sakaluk). 
+
+{{< figure src = "speakers.png" class = "center" width = "600" alt = "Headshots of the moderator and four panelists">}}
+
+We decided to write this additional blog post to provide a ready reference and description of this community call, which introduced the system being developed for peer review of explicitly statistical software, along with a couple of the automated software tools for use by developers and reviewers of statistical software.
```
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:32]:
```suggestion
This post provides a ready reference and description of this community call, which introduced the system being developed for peer review of explicitly statistical software, along with a couple of the automated software tools for use by developers and reviewers of statistical software.
```

File: content/blog/2021-03-09-commcall-stats/index.md (hunk 4 above)
Comment (stefaniebutland) [inline content/blog/2021-03-09-commcall-stats/index.md:44]:
```suggestion
Here we've organized the video content by speakers and questions, including links to the specific time points in the [video](https://vimeo.com/518761488) as well as to questions and answers in the [collaborative notes document](https://docs.google.com/document/d/1o933nG1ZW2Qf8p7FdQxn8wOkk9qoBINL1fx3FlIFSJU/). 
```

**Review by stefaniebutland — COMMENTED** [review]
I've made some suggestions inline, plus these below, and I see a couple from Noam. @steffilazerte pls ping me when you're done and I'll contact speakers for approval

- @steffilazerte please fix headshots image - white background shows for some (Chrome browser)
Screen Shot 2021-03-08 at 9 20 43 AM
- pull quote
  - attribute to John Sakaluk
  - edit out the "thinking out loud" text like "kind of" & others to tighten, and John can approve the final quote

**Comment by stefaniebutland** [comment]
For tweets, I am super-tempted to have a thread, with one tweet per question asked by audience. No need to tag the asker. It gives a window to the richness of specific content. 
You cool with that @steffilazerte ?

**Comment by steffilazerte** [comment]
Hi @stefaniebutland I've made the changes let me know what you think. Good catch on the backgrounds, it took me a while to figure out what it was you wanted changed (only saw it when I tilted my head at a certain angle!). 

I'm not convinced that tweeting the questions will be useful. A lot of the questions are a) not really understandable without reading the summary post at least (they directly refer to srr or autotest) or b) too specific to really ask well in a tweet. 

If you really want the questions in tweets, perhaps I could include more of a summary of the post in a tweet thread, explaining what autotest and srr are and then cherry pick a couple of the more straightforward questions to include.

**Comment by stefaniebutland** [comment]
> perhaps I could include more of a summary of the post in a tweet thread, explaining what autotest and srr are and then cherry pick a couple of the more straightforward questions to include.

This is more of what I was thinking yes. My thought is that the title of the comm call does not tell people what it was really about, and neither does the summary text (by necessity of length). So a tweet thread highlights the content to a wider audience.

@steffilazerte If it won't take too long or too much mental energy, please do that. Ok to go with planned tweets otherwise!

**Pull request merged**